integration-cli/docker_cli_plugins_logdriver_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/docker/docker/integration-cli/checker"
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) TestPluginLogDriver(c *check.C) {
+	testRequires(c, IsAmd64, DaemonIsLinux)
+
+	pluginName := "cpuguy83/docker-logdriver-test:latest"
+
+	dockerCmd(c, "plugin", "install", pluginName)
+	dockerCmd(c, "run", "--log-driver", pluginName, "--name=test", "busybox", "echo", "hello")
```

Comment:
I expect startup/shutdown times to be slightly longer with a plugin since it has to go over the plugin sock to signal it to start logging (and subsequently stop logging).

Log streaming from the container should be faster (in terms of making it out of the daemon) since the `jsonfilelog` logger is comparatively less efficient than the plugin stream.

Since my plugin is only wrapping the `jsonfilelog` this will be a double hit because it has to marshal the proto and then (in plugin) unmarshal > marshal to json > write to disk.